hg-fast-export.py

```diff
@@ -139,6 +139,25 @@ def get_author(logmessage,committer,authors):
       return r
   return committer
 
+def git_remote(path):
```

Comment:
Can't Python's `configparser` understand Git's config format?

Reply:
Unfortunately, no. It fails to read it properly

Reply:
Still on the fence on this, but wouldn't it be better to just fork a git command to ask about this?